### ISSUE TITLE
240hz+OD turn off after every reboot

### ISSUE THREAD

**Issue by ComplicatedDoc** [seerge/g-helper#3292]
### Rules

- [X] I made myself familiar with the <a href='https://github.com/seerge/g-helper?tab=readme-ov-file'>Readme</a>, <a href='https://github.com/seerge/g-helper/wiki/FAQ'>FAQ</a> and <a href='https://github.com/seerge/g-helper/wiki/Troubleshooting'>Troubleshooting</a>.
- [X] I understand that, if insufficient information or no app logs will be provided, my issue will be closed without an answer.

### What's wrong?

i have asus rog strix scar 18 2024 , after every reboot the overdrive settings turn off...i have to turn everytime manual 240hz+OD
i installed the g-helper the correct way, i uninstalled everything from asus, and i turned off every asus services

### How to reproduce the bug?

i have asus rog strix scar 18 2024 , after every reboot the overdrive settings turn off...i have to turn everytime manual 240hz+OD
i installed the g-helper the correct way, i uninstalled everything from asus, and i turned off every asus services

### Logs

[log.txt](https://github.com/user-attachments/files/17495926/log.txt)

### Device and Model

ROG STRIX SCAR 18

### Armoury Crate

Uninstalled

### Asus Services

0

### Version

last one

### OS

windows 11 23H2

**Comment by seerge** [comment]
@ComplicatedDoc hello,

According to the log app sets Overdrive flag every single time it starts on boot.
```
23/10/2024 20:17:05: ------------
23/10/2024 20:17:05: App launched: ROG Strix SCAR 18 G834JYR_G834JYR :0.194.0.0en-GB
23/10/2024 20:17:05: Stopped: ASUSSmartDisplayControl
....
23/10/2024 20:17:07: BatteryLimit = 70 : OK
23/10/2024 20:17:07: Miniled2 = 1 : OK
23/10/2024 20:17:07: ScreenOverdrive = 1 : OK <---- OVERDRIVE IS SET ON START
```

How do you know that it is not enabled ? I have also noticed that you have `ASUSSmartDisplayControl` app installed, try to uninstall it as it may interfere

**Comment by ComplicatedDoc** [comment]
because after every reboot its showing me just 240hz, not 240hz+OD, after i press on 240hz, then show 240hz+overdrive

**Comment by ComplicatedDoc** [comment]
i will uninstall display control

**Comment by ComplicatedDoc** [comment]
yes, all good now, thanks, i uninstall asus smart display control, now the display remain 240hz+OD. thanks!!:)

**Comment by seerge** [comment]
@ComplicatedDoc great
